UI/data/locale/en-US.ini

```diff
@@ -1057,7 +1057,34 @@ Screenshot.Source="Screenshot (Source)"
 FilenameFormatting.completer="%CCYY-%MM-%DD %hh-%mm-%ss\n%YY-%MM-%DD %hh-%mm-%ss\n%Y-%m-%d %H-%M-%S\n%y-%m-%d %H-%M-%S\n%a %Y-%m-%d %H-%M-%S\n%A %Y-%m-%d %H-%M-%S\n%Y-%b-%d %H-%M-%S\n%Y-%B-%d %H-%M-%S\n%Y-%m-%d %I-%M-%S-%p\n%Y-%m-%d %H-%M-%S-%z\n%Y-%m-%d %H-%M-%S-%Z\n%FPS\n%CRES\n%ORES\n%VF"
 
 # basic mode 'output' settings - advanced section - recording subsection - TT
-FilenameFormatting.TT="%CCYY	Year, four digits\n%YY		Year, last two digits (00-99)\n%MM		Month as a decimal number (01-12)\n%DD		Day of the month, zero-padded (01-31)\n%hh		Hour in 24h format (00-23)\n%mm		Minute (00-59)\n%ss		Second (00-61)\n%%		A % sign\n%a		Abbreviated weekday name\n%A		Full weekday name\n%b		Abbreviated month name\n%B		Full month name\n%d		Day of the month, zero-padded (01-31)\n%H		Hour in 24h format (00-23)\n%I		Hour in 12h format (01-12)\n%m		Month as a decimal number (01-12)\n%M		Minute (00-59)\n%p		AM or PM designation\n%s		Time in seconds since UNIX epoch\n%S		Second (00-61)\n%y		Year, last two digits (00-99)\n%Y		Year\n%z		ISO 8601 offset from UTC in timezone\n%Z		Timezone name or abbreviation\n%FPS		Frames per second\n%CRES		Base (canvas) resolution\n%ORES		Output (scaled) resolution\n%VF		Video format"
+FilenameFormatting.TT.CCYY="Year, four digits"
+FilenameFormatting.TT.YY.year="Year, last two digits (00-99)"
+FilenameFormatting.TT.MM.month="Month as a decimal number (01-12)"
+FilenameFormatting.TT.DD.day="Day of the month, zero-padded (01-31)"
+FilenameFormatting.TT.hh="Hour in 24h format (00-23)" # fail
+FilenameFormatting.TT.mm.minute="Minute (00-59)"
+FilenameFormatting.TT.ss="Second (00-61)"
+FilenameFormatting.TT.%="A % sign"
```

Comment:
For safety, I'd probably use the word "Percent" rather than "%" in the key.

Reply:
I've revised it to `FilenameFormatting.TT.Percent`.
Also removed some comments ` # fail`, which was added while debugging the case-sensitivity bug and should not be left in the file.